### ISSUE TITLE
Feature Request: Orb Inventory, Legend Materia list, Soul Break Tier/Realm view

### ISSUE THREAD

**Issue by BaconCatBug** [rk-squared/rk-squared#2]
Feature Request:
Orb inventory with "Major Orb Equivalent"

A Legend Materia sub-tab under Record Materia.

The ability to view Soul Breaks listed by tier and realm.

**Comment by joshkel** [comment]
Thanks for the feedback. I'll work toward including these, although I don't always have the time to do everything I'd want.

Legend Materia sub-tab: What are you envisioning here? Legend materia already show under soul breaks. I like the Record Materia view because it organizes by category, but that doesn't seem too applicable to legend breaks (since they can't be shared between characters). Or is it?

**Comment by BaconCatBug** [comment]
Yeah that's my bad I forgot that legend materia were in the soul break tab.
Please ignore my idiocy.
